rulesets/NI.TestUtilities.ruleset

```diff
@@ -9,13 +9,10 @@
     <Rule Id="CA1008" Action="None" />
     <Rule Id="CA1017" Action="None" />
     <Rule Id="CA1018" Action="None" />
-    <Rule Id="CA1019" Action="Warning" />
```

Comment:
Things like this where the TestUtilities/Tests rulesets were more restrictive than the base are wrong. I've just deleted them. Either they differed or were duplicated but had the same action.

Reply:
Are you sure none of these should be on in the base?

Reply:
They might be, but I am first getting this repo's files in sync with asw and then I will do another scrub on each override in the tests and utilities. That's coming this week.